### ISSUE TITLE
Inconsistency between element numbering in GUI and code

### ISSUE THREAD

**Issue by tamirlance** [NREL/SolTrace#17]
Element numbering in scripting language starts at 0 but in GUI it starts at 1 which can be sometimes confusing

**Comment by mjwagner2** [comment]
Good point. I suggest that the fix should be to correct the numbering in the GUI to start at 0.
